src/processors/models/processorsManager.js

```diff
@@ -62,8 +64,10 @@ module.exports.updateProcessor = async function (processorId, processor) {
     }
 
     processor.created_at = oldProcessor.created_at;
-    fileManager.validateJavascriptContent(processor.javascript);
+    let exportedFunctions = verifyJSAndGetExportedFunctions(processor.javascript);
+    processor.exported_functions = exportedFunctions;
     await databaseConnector.updateProcessor(processorId, processor);
+    processor.exported_functions = verifyJSAndGetExportedFunctions(processor.javascript, true);
```

Comment:
This isn't needed again. And `verifyJSAndGetExportedFunctions` is accepting only 1 argument in the signature and here you are passing 2